python/xorbits/pandas/mars_adapters/tests/test_execution.py

```diff
@@ -70,3 +72,31 @@ def test_from_mars_execution_condition(setup, dummy_df, dummy_str_series):
         # TODO: series falling back methods not implemented yet.
         # assert not need_to_execute(dummy_str_series.to_pickle(os.path.join(tempdir, "series.pkl")))
         # assert not need_to_execute(dummy_str_series.to_json())
+
+
+def test_own_data(setup, dummy_df, dummy_str_series):
```

Comment:
Better add some tests to check if the docstring works.

Reply:
Added.